benchmark/redisgraph/README.md

```diff
@@ -0,0 +1,267 @@
+
+
+## Current use cases
+This article documents the details on how to reproduce the graph database benchmark result on RedisGraph.
+Currently, RedisGraph benchmark supports two use cases:
+
+
+Name | Description and Source | Vertices | Edges
+-- | -- | -- | --
+graph500 | Synthetic Kronecker graph http://graph500.org | 2.4 M | 64 M
+twitter | Twitter user-follower directed graph http://an.kaist.ac.kr/traces/WWW2010.html | 41.6 M | 1.47 B                                                                                                                                                                                                 
+
+### How to use the benchmark
+
+Graph benchmarking involves 3 phases: data retrieval, data loading/insertion, and query execution.
+
+## Installation
+The easiest way to get and install the benchmark code is to use:
+```bash
+git clone https://github.com/RedisGraph/graph-database-benchmark.git
+cd graph-database-benchmark/benchmark/redisgraph
+sudo pip install -r requirements.txt
+```
+
+#### Data retrieval 
+
+Variables:
+1. `DATABASE_HOST` (default: `127.0.0.1`)
+1. `DATABASE_PORT` (default: `6379`)
+1. `BULK_DATA_DIR` (default: `/tmp/bulk_data`)
+1. `DATASET_NAME` (default: `dependent on the use case`)
+1. `EDGE_FILE` (default: `dependent on the use case`)
+1. `NODE_FILE` (default: `dependent on the use case`)
+
+The easiest way to get the datasets required for the benchmark is to use one of the helper scrips:
+1. `./scripts/get_graph500_dataset.sh`
+or
+1. `./scripts/get_twitter_dataset.sh`
+
+Sample output:
+```bash
+./scripts/get_graph500_dataset.sh
+```
+
+### Data insertion
+
```

Comment:
I'd put a note here that the server should be live prior to running these scripts! Consider adding a quick ping at the top of these scripts to exit immediately otherwise, in fact (not mandatory, but a bit of a time-saver).